frontend/packages/operator-lifecycle-manager/src/types.ts

```diff
@@ -200,6 +200,9 @@ export type CatalogSourceKind = {
 export type PackageManifestKind = {
   apiVersion: 'packages.operators.coreos.com/v1';
   kind: 'PackageManifest';
+  metadata: {
+    tags?: string[];
```

Comment:
@alecmerdler Is this right? I don't actually see this in the response, but it's used here:

https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx#L75

Reply:
No I don't think that is a real field.